pkg/cmd/server/bootstrappolicy/securitycontextconstraints.go

```diff
@@ -296,30 +295,6 @@ func GetBootstrapSecurityContextConstraints(sccNameToAdditionalGroups map[string
 			panic(err)
 		}
 		constraints[i] = constraint
-
-		if usersToAdd, ok := sccNameToAdditionalUsers[constraint.Name]; ok {
-			constraints[i].Users = append(constraints[i].Users, usersToAdd...)
-		}
-		if groupsToAdd, ok := sccNameToAdditionalGroups[constraint.Name]; ok {
-			constraints[i].Groups = append(constraints[i].Groups, groupsToAdd...)
-		}
 	}
 	return constraints
 }
-
-// GetBoostrapSCCAccess provides the default set of access that should be passed to GetBootstrapSecurityContextConstraints.
-func GetBoostrapSCCAccess(infraNamespace string) (map[string][]string, map[string][]string) {
```

Comment:
General question: what of this still needs this access.  If it still needs this access, can the access be namespace scoped and/or managed by CVO manifest (i.e. moved out of origin).